File: BackEnd/ajax/Hospitalizacion.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Hospitalizacion.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'idPiso': null,
    'idMedico': null,
    'fechaHoraAlta': null,
    'fechaHoraIngreso': null,
    'observacion': null,
    'idExpediente': null,
    'cama': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Hospitalizacion.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPiso',
    'idPiso': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Hospitalizacion.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Hospitalizacion.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorMedico',
    'idCentroMedico': null,
    'idMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Hospitalizacion.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'idPiso': null,
    'idIngreso': null,
    'idMedico': null,
    'fechaHoraIngreso': null,
    'observacion': null,
    'cama': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Hospitalizacion.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorFecha',
    'idCentroMedico': null,
    'fechaHoraIngreso': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Hospitalizacion.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'darAlta',
    'fechaHoraAlta': null,
    'idIngreso': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Hospitalizacion.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentro',
    'idCentroMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

File: BackEnd/postman/Hospitalizacion.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Hospitalizacion'
data={
  'accion':'crear',
  'idPiso': None,
  'idMedico': None,
  'fechaHoraAlta': None,
  'fechaHoraIngreso': None,
  'observacion': None,
  'idExpediente': None,
  'cama': None,
}
print postear(data,url)
print

url='Hospitalizacion'
data={
  'accion':'listarPorPiso',
  'idPiso': None,
}
print postear(data,url)
print

url='Hospitalizacion'
data={
  'accion':'listarPorPaciente',
  'idExpediente': None,
}
print postear(data,url)
print

url='Hospitalizacion'
data={
  'accion':'listarPorMedico',
  'idCentroMedico': None,
  'idMedico': None,
}
print postear(data,url)
print

url='Hospitalizacion'
data={
  'accion':'actualizar',
  'idPiso': None,
  'idIngreso': None,
  'idMedico': None,
  'fechaHoraIngreso': None,
  'observacion': None,
  'cama': None,
}
print postear(data,url)
print

url='Hospitalizacion'
data={
  'accion':'listarPorFecha',
  'idCentroMedico': None,
  'fechaHoraIngreso': None,
}
print postear(data,url)
print

url='Hospitalizacion'
data={
  'accion':'darAlta',
  'fechaHoraAlta': None,
  'idIngreso': None,
}
print postear(data,url)
print

url='Hospitalizacion'
data={
  'accion':'listarPorCentro',
  'idCentroMedico': None,
}
print postear(data,url)
print
